pkg/kubebench/scanner.go

```diff
@@ -121,7 +121,7 @@ func (s *Scanner) Scan(ctx context.Context, node corev1.Node) (v1alpha1.CISKubeB
 		Report: output,
 	}
 
-	err = controllerutil.SetOwnerReference(&node, &report, s.scheme)
+	err = controllerutil.SetControllerReference(&node, &report, s.scheme)
```

Comment:
By changing owner to controller we can take advantage of controller-runtime's controller builder. Such builder can watch the node, but also owned report. Hence, whenever a CISKubeBenchReport is deleted a reconciliation for the owning node is triggered.